src/m.ts

```diff
@@ -0,0 +1,1294 @@
+      name: "info",
+      description: "Show system information",
+    },
+    {
+      name: "itunes",
+      description: "Manage iTunes",
+      subcommands: [
+        {
+          name: "status",
+          description: "Show status",
+        },
+        {
+          name: "play",
+          description: "Play track",
+        },
+        {
+          name: "pause",
+          description: "Pause track",
+        },
+        {
+          name: "next",
+          description: "Play next track",
+        },
+        {
+          name: "prev",
+          description: "Play previous track",
+        },
+        {
+          name: "mute",
+          description: "Mute iTunes",
+        },
+        {
+          name: "unmute",
+          description: "Unmute iTunes",
+        },
+        {
+          name: "vol",
+          description: "Volume",
+          args: {
+            name: "up/down/#",
+            description: "Up/Down/Number",
+            suggestions: ["up", "down" /*, "#",*/],
+            isOptional: true,
+          },
+        },
+        {
+          name: "stop",
+          description: "Stop track",
+        },
+        {
+          name: "quit",
+          description: "Quit iTunes",
+        },
+      ],
+    },
+    {
+      name: "localhost",
+      description: "Manage localhost",
+      subcommands: [
+        {
+          name: "ls",
+          description: "List current records in localhost",
+        },
+        {
+          name: "add",
+          description: "Add a new host to the localhost file",
+          args: [
+            {
+              name: "IP",
+              description: "IP of the host",
+            },
+            {
+              name: "HOSTNAME",
+              description: "Hostname of the host",
+            },
+          ],
+        },
+        {
+          name: "remove",
+          description: "Remove a host from the localhost file",
+          args: {
+            name: "HOSTNAME",
+            description: "Hostname of the host",
+          },
+        },
+      ],
+    },
+    {
+      name: "lock",
+      description: "Lock session",
+    },
+    {
+      name: "network",
+      description: "Manage network",
+      subcommands: [
+        {
+          name: "ls",
+          description: "List network interfaces",
+        },
+        {
+          name: "location",
+          description: "Manage location",
+          subcommands: [
+            {
+              name: "ls",
+              description: "List locations",
+            },
+            {
+              name: "create",
+              description: "Create a location",
+              args: {
+                name: "name",
+                description: "Name of the location",
+              },
+            },
+            {
+              name: "delete",
+              description: "Delete a location",
+              args: {
+                name: "name",
+                description: "Name of the location",
+              },
```

Comment:
Can make a generator:
```ts
const generateNetworkLocations: Fig.Generator = {
  script: "m network list | grep -e 'Device: ' | cut -d' ' -f2-",
  splitOn: "\n",
};
```

```suggestion
              args: {
                name: "name",
                description: "Name of the location",
                generators: generateNetworkLocations,
              },
```

And you can add this to the `switch` subcommand below